Android/app/src/main/java/io/github/project_travel_mate/utilities/BugReportFragment.java

```diff
@@ -0,0 +1,63 @@
+package io.github.project_travel_mate.utilities;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
+
+import io.github.project_travel_mate.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link BugReportFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class BugReportFragment extends Fragment implements AdapterView.OnItemSelectedListener {
+
+    public BugReportFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment.
+     *
+     * @return A new instance of fragment BugReportFragment.
+     */
+    public static BugReportFragment newInstance() {
+        BugReportFragment fragment = new BugReportFragment();
+        return fragment;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_bug_report, container, false);
+
+        Spinner bugTypeSpinner = view.findViewById(R.id.spinner_bug_type);
+        ArrayAdapter<CharSequence> spinnerAdapter = ArrayAdapter.createFromResource(getActivity(),
+                R.array.bug_types, android.R.layout.simple_spinner_item);
+
+        bugTypeSpinner.setOnItemSelectedListener(this);
+        spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        bugTypeSpinner.setAdapter(spinnerAdapter);
+
+        return view;
+    }
+
+    @Override
+    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+        Log.d("bugtype", "" + parent.getItemAtPosition(position).toString());
```

Comment:
We will be making the API call here?

Reply:
Not here, I'll make it on button click. I haven't initialised edit-text nor button.

Reply:
Just create the appropriate variables for bug type & message, and assign them the values.
We'll add the API call later.

Reply:
Do that also, everything should be ready other than the call part.

Reply:
Sure, will do it after a couple of hours. It's past midnight here.

Reply:
haha! Sure. no problem 😆